Scripts/detect_public_api_changes/package_file_helper.swift

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env xcrun swift
```

Comment:
I suggest:
- adding default output listing all supported CLI parameters if none is provided

Reply:
I love these ideas :)
Already tackled that in the [next version](https://github.com/Adyen/adyen-ios/pull/1718)

Reply:
🤣 I feel now I didn't spent my time wisely reviewing this one alone